service/refund_test.go

```diff
@@ -38,7 +38,7 @@ func TestUnitCreateRefund(t *testing.T) {
 			GetGovPayRefundSummary(req, id).
 			Return(nil, nil, Error, err)
 
-		refund, status, err := service.CreateRefund(req, id, body)
+		_, refund, status, err := service.CreateRefund(req, id, body)
```

Comment:
if paymentSession is now being returned from the service, the unit tests should assert this value at least once.